source/server/admin/prometheus_stats.cc

```diff
@@ -26,6 +38,20 @@ std::string sanitizeName(const absl::string_view name) {
   return promRegex().replaceAll(name, "_");
 }
 
+/**
+ * Take tag values and sanitize it for text serialization, according to
+ * Prometheus conventions.
+ */
+std::string sanitizeValue(const absl::string_view value) {
```

Comment:
Could you rename this to be more accurate one like `sanitizeTagName`?

Reply:
Did you mean `sanitizeTagValue`? `sanitizeTagName` does not seem to be appropriate for tag value.

On the other hand, renaming this function will result in inconsistency in names: `sanitizeName` and `sanitizeTagValue`.

Reply:
@mathetake as @jewertow mentions, I tried to make the function name parallel the other sanitization function name. It is handling tag values. Would you prefer `sanitizeTagValue` ?

Reply:
OK I got it! Thank you for thinking about it